assets/sass/patterns/molecules/popup.scss

```diff
@@ -6,7 +6,8 @@
 
 .popup {
   position: absolute;
-  width: 340px;
+  min-width: 340px;
+  max-width: #{$bkpt-site--medium - 10 - 10}px; // threshold - left - right
```

Comment:
The text in the about profile popups can be quite long, so this makes them bigger. This also changes the other popups, which might not be desirable.